src/group_theory/subgroup/basic.lean

```diff
@@ -1030,6 +1030,26 @@ instance bot_normal : normal (⊥ : subgroup G) := ⟨by simp⟩
 @[priority 100, to_additive]
 instance top_normal : normal (⊤ : subgroup G) := ⟨λ _ _, mem_top⟩
 
+/-- Group conjugation on a normal subgroup. Analogous to `mul_aut.conj`. -/
+def mul_aut.conj_normal [hH : H.normal] : G →* mul_aut H :=
```

Comment:
If #8627 goes through first, this might make sense to instead provide as `mul_distrib_mul_action (conj G) H`

Reply:
I could add this, but I would still rather keep the direct analog of `mul_aut.conj`.

Reply:
The direct analog is `mul_distrib_mul_action.to_mul_aut`